.travis.yml

```diff
@@ -1,17 +1,13 @@
-sudo: false
-dist: trusty
 language: rust
-
-before_install:
-  - cargo install mdbook --vers '0.0.25' --debug --force
-
+cache:
```

Comment:
Can you get rid of the cache like I mentioned in the other comment?

Reply:
See above. If it works now, it might not hurt leaving in cache. :-) Otherwise I'll remove it.